### ISSUE TITLE
`drop-path` rate for ImageNet-21k 90 epochs training.

### ISSUE THREAD

**Issue by bhheo** [facebookresearch/deit#176]
Hi

Thank you for your effort on an excellent code base. 
I think it is the best practice code for ViT and IN-21k training.

I'm trying DeiT-III on ImageNet-21k dataset.
I have a question about `drop-path` rate

In Table 5 of DeiT-III paper, it is mentioned that `drop-path` rate increased by 0.05 for the longer pre-training (240 epochs).
In Table 13,  `drop-path` rates for IN-21k 90 epochs training are `0.0, 0.1, 0.3, 0.5` for ViT-S, B, L, H, respectively.
So, I thought 
- 90 epochs : ViT-S, B, L, H `0.0, 0.1, 0.3, 0.5`
- 240 epochs : ViT-S, B, L, H `0.05, 0.15, 0.35, 0.55`

However, in example code, you used 
- 240 epochs : ViT-S, B, L, H `0.05, 0.1, 0.3, 0.5`

Then, 90 epochs setting should be
- 90 epochs : ViT-S, B, L, H `0.0, 0.05, 0.25, 0.45`

So, I'm confused and want to ask for some advice on it.
In short, which setting should I use for 90 epochs IN-21k training for the best results?
1. 90 epochs : ViT-S, B, L, H `0.0, 0.05, 0.25, 0.45`
2. 90 epochs : ViT-S, B, L, H `0.0, 0.1, 0.3, 0.5`

**Comment by tangjiasheng** [comment]
hi, @bhheo 

I just wanna know, which version of 21k do you use and how do you do train-test split?
Much appreciate for your kind share.

Best,
Jiasheng

**Comment by bhheo** [comment]
Hi @tangjiasheng 

I downloaded IN21k from the official image-net website.
I will try `winter21` without validation set and `imagenet21k_resized` with train-test split on ImageNet official sites.
But, I don't have the result yet and I'm not sure it is the correct setting.

Best
Byeongho Heo

**Comment by tangjiasheng** [comment]
@bhheo 
I also have a same concern, not sure which setting is right for reproducing. But, as for 21k, I don't think it will lead too much gap of the results. Unfortunately, I cannot get similar result. 

Best,
Jiasheng

**Comment by TouvronHugo** [comment]
Hi @bhheo,

Thank you for your message. 
There is an error in the paper. The drop-path of the github repo are correct. 
You can use the same drop-path for 90 epoch and 240 epoch.

For the 21k logs we evaluate the model on the 1k val, since the 1k labels are included in the 21k labels it is possible to make the prediction without modifying the network.

Best,

Hugo

**Comment by bhheo** [comment]
Thanks you
